qibullet/simulation_manager.py

```diff
@@ -54,7 +54,10 @@ def launchSimulation(
         """
         if gui:  # pragma: no cover
             physics_client = pybullet.connect(pybullet.GUI)
-            pybullet.setRealTimeSimulation(1, physicsClientId=physics_client)
+
+            if auto_step:
+                pybullet.setRealTimeSimulation(1, physicsClientId=physics_client)
```

Comment:
Please use the following format:
```python
pybullet.setRealTimeSimulation(
    1,
    physicsClientId=physics_client)
```
to respect the 80 characters limit

Reply:
ok